src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -3239,7 +3239,83 @@ class SetRemoveElementError extends SetRemoveElementResponse
     use ErrorBody;
 }
 
-// placeholder: sortedSetLengthByScore
+/**
+ * Parent response type for a sorted set length by score request. The
+ * response object is resolved to a type-safe object of one of
+ * the following subtypes:
+ *
+ * * SortedSetLengthByScoreSuccess
+ * * SortedSetLengthByScoreError
+ *
+ * Pattern matching can be used to operate on the appropriate subtype.
+ * For example:
+ * <code>
+ * if ($success = $response->asSuccess()) {
+ *     return $success->length();
+ * } elseif ($error = $response->asError())
+ *     // handle error as appropriate
+ * }
+ * </code>
+ */
+abstract class SortedSetLengthByScoreResponse extends ResponseBase
+{
+    /**
+     * @return SortedSetLengthByScoreSuccess|null Returns the success subtype if the request was successful and null otherwise.
+     */
+    public function asSuccess(): ?SortedSetLengthByScoreSuccess
+    {
+        if ($this->isSuccess()) {
+            return $this;
+        }
+        return null;
+    }
+
+    /**
+     * @return SortedSetLengthByScoreError|null Returns the error subtype if the request returned an error and null otherwise.
+     */
+    public function asError(): ?SortedSetLengthByScoreError
+    {
+        if ($this->isError()) {
+            return $this;
+        }
+        return null;
+    }
+}
+
+/**
+ * Indicates that the request that generated it was successful.
+ */
+class SortedSetLengthByScoreSuccess extends SortedSetLengthByScoreResponse
+{
+    private int $length;
+
+    public function __construct(_SortedSetLengthByScoreResponse $response)
+    {
+        parent::__construct();
+        $this->length = $response->getFound() ? $response->getFound()->getLength() : 0;
+    }
```

Comment:
Just leaving a note: the "or 0" case should never happen. I opted to keep the protobuf responses out of here and pass concrete values to here from the data client.